core/src/main/java/com/scalar/db/storage/objectstorage/ObjectStorageProvider.java

```diff
@@ -0,0 +1,20 @@
+package com.scalar.db.storage.objectstorage;
+
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.api.DistributedStorageProvider;
+import com.scalar.db.common.CommonDistributedStorageAdmin;
+import com.scalar.db.config.DatabaseConfig;
+
+public interface ObjectStorageProvider extends DistributedStorageProvider {
```

Comment:
[minor] There are already `XxxxxxFactory` classes. How about `ObjectStorageFactory` and `DistributedStorageFactory` to align with them?